### PR TITLE
fix(publishing): add preval pino version

### PR DESCRIPTION
**Pull request by budarin**
Preval pino version before publishing the package in the prepare package stage

### CODE REVIEW

**Comment by mcollina** [comment]
the approach we follow in Fastify is slightly different

There is https://github.com/fastify/fastify/blob/c37888c281087089c37b3e24f2d2fa61a3304ce4/package.json#L19 that verifies that the version in meta is the same of package.json.

We manually bump the version by running the update script (no prepare step).

**Comment by budarin** [comment]
ок,I'll refactor the code

File: lib/utils/preval-pino-version.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const fs = require('fs')
```
Comment (mcollina) [inline lib/utils/preval-pino-version.js:1]: move this file to the build/ folder

**Review by mcollina — CHANGES_REQUESTED** [review]
Please take a look at how it's done in Fastify. This is still far from it.

There is no package.json script but a test to verify if the meta version number has been bumped.

**Review by mcollina — CHANGES_REQUESTED** [review]
Could you add https://github.com/fastify/fastify/blob/1152a45f433b05b56a9c4206855d1322cecb6e83/package.json#L19

and 

https://github.com/fastify/fastify/blob/main/test/internals/version.test.js

**Review by mcollina — APPROVED** [review]
lgtm, thanks

File: test/internals/version.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const t = require('tap')
+const test = t.test
+const fastify = require('../..')()
```
Comment (climba03003) [inline test/internals/version.test.js:7]:
```suggestion
const pino = require('../..')()
```

File: test/internals/version.test.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+test('should be the same as package.json', t => {
+  t.plan(1)
+
+  const json = JSON.parse(fs.readFileSync(path.join(__dirname, '..', '..', 'package.json')).toString('utf8'))
+
+  t.equal(fastify.version, json.version)
```
Comment (climba03003) [inline test/internals/version.test.js:14]:
```suggestion
  t.equal(pino.version, json.version)
```

**Pull request merged**